### ISSUE TITLE
Info returned by `--version` has flaws

### ISSUE THREAD

**Issue by KurtPfeifle** [AppImage/AppImageUpdate#62]
Current binary releases return version info with "Schönheitsfehlers":

1. **AppImageUpdate:**

    ```
    $> ./AppImageUpdate-185-1c72f3b-x86_64.AppImage --version 
       AppImageUpdate version 1-alpha (commit 1c72f3b), build 185 built on 2018-01-03 16:41:48 UTC
       AppImageUpdate version 1-alpha (commit 1c72f3b), build 185 built on 2018-01-03 16:41:48 UTC
    ```

    AppImageUpdate prints *two* (identical) lines instead of only *one*.

1. **appimageupdatetool:**

    ```
    $> ./appimageupdatetool-185-1c72f3b-x86_64.AppImage --version
       AppImageUpdate version 1-alpha (commit 1c72f3b), build 185 built on 2018-01-03 16:41:48 UTC
    ```

    appimageupdatetool returns a single line, but announces itself as the GUI (*'AppImageUpdate'* instead of the CLI name, *'appimageupdatetool'*).

**Comment by probonopd** [comment]
re 1. @TheAssassin I think that `--version` does not need to do anything because the version is already printed in `main()`.

**Comment by TheAssassin** [comment]
Fixed by 8cf6294a87959b9adabbee28d2fe723aef42060e and 6e634ca34249afee4fb500bda0df2a49bf08e5d3.
